pkg/platform/model/vcs.go

```diff
@@ -643,6 +600,34 @@ func CommitInitial(hostPlatform string, langName, langVersion string) (strfmt.UU
 	return res.Payload.CommitID, nil
 }
 
+func versionStringToConstraints(version string) ([]*mono_models.Constraint, error) {
+	constraints := []*mono_models.Constraint{}
```

Comment:
Note that exact versions (no wildcards) will be supported in a separate PR that addresses DX-2010.